package/MDAnalysis/analysis/pca.py

```diff
@@ -163,6 +167,7 @@ class PCA(AnalysisBase):
     Computation can be speed up by supplying a precalculated mean structure
 
     .. versionchanged:: 1.0.0
```

Comment:
Also add a versionchanged entry for correct cumulative variance calculation for custom n_components.